corehq/apps/sofabed/migrations/0010_remove_like_indexes.py

```diff
@@ -0,0 +1,106 @@
+# encoding: utf-8
+import datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        
+        # Removing index on 'CaseActionData', fields ['action_type']
+        db.execute("DROP INDEX IF EXISTS sofabed_caseactiondata_action_type_like")
+        db.execute("DROP INDEX IF EXISTS sofabed_caseactiondata_case_id_like")
+        db.execute("DROP INDEX IF EXISTS sofabed_caseactiondata_user_id_like")
+
+        db.commit_transaction()
+        db.start_transaction()
+
+        db.execute("DROP INDEX IF EXISTS sofabed_casedata_case_id_like")
+        db.execute("DROP INDEX IF EXISTS sofabed_casedata_closed_by_like")
+        db.execute("DROP INDEX IF EXISTS sofabed_casedata_doc_type_like")
+        db.execute("DROP INDEX IF EXISTS sofabed_casedata_domain_like")
+        db.execute("DROP INDEX IF EXISTS sofabed_casedata_modified_by_like")
+        db.execute("DROP INDEX IF EXISTS sofabed_casedata_opened_by_like")
+        db.execute("DROP INDEX IF EXISTS sofabed_casedata_owner_id_like")
+        db.execute("DROP INDEX IF EXISTS sofabed_casedata_type_like")
+        db.execute("DROP INDEX IF EXISTS sofabed_casedata_user_id_like")
+        db.execute("DROP INDEX IF EXISTS sofabed_casedata_version_like")
+
+        db.commit_transaction()
+        db.start_transaction()
+
+        db.execute("DROP INDEX IF EXISTS sofabed_caseindexdata_case_id_like")
+        db.execute("DROP INDEX IF EXISTS sofabed_caseindexdata_identifier_like")
+        db.execute("DROP INDEX IF EXISTS sofabed_caseindexdata_referenced_type_like")
```

Comment:
We don't need any of these queries since we don't do any partial matching queries.

I've tested these and they will be quick to run.